tests.toml

```diff
@@ -0,0 +1,9 @@
+test_format = 1.0
+
+[default]
+
+    exclude = [
+        "install.nourl"
+    ]
```

Comment:
I'm not sure this is needed, package check should automatically understand that this is a webapp and not run this test

```suggestion
```